src/main/java/com/qaprosoft/carina/demo/utils/AuthService.java

```diff
@@ -0,0 +1,43 @@
+package com.qaprosoft.carina.demo.utils;
+
+import com.qaprosoft.carina.demo.gui.pages.AutomationPracticeCreateAccountPage;
+import com.qaprosoft.carina.demo.gui.pages.AutomationPracticeSignInPage;
+
+import java.util.Random;
+
+public class AuthService {
+
+    public void signIn(AutomationPracticeSignInPage signInPage, String email, String password) {
+        signInPage.typeEmail(email);
+        signInPage.typePassword(password);
+        signInPage.clickSignInButton();
+    }
+
+    public void signIn(AutomationPracticeSignInPage signInPage) {
```

Comment:
I like this method with randomly generated email and password. Imagine that you need to use this email and password somewhere else in the test. I suggest to return email and password to the test